MsCorePkg/Library/PlatformBootManagerLib/BdsPlatform.c

```diff
@@ -236,34 +236,24 @@ PlatformBootManagerBeforeConsole (
     FreePool (ConsoleOut);
   }
 
-  //
-  // Fill ConIn/ConOut in Full Configuration boot mode
-  //
-  DEBUG ((DEBUG_INFO, "%a - %x\n", __FUNCTION__, mBootMode));
-  if ((mBootMode == BOOT_WITH_FULL_CONFIGURATION) ||
-      (mBootMode == BOOT_WITH_DEFAULT_SETTINGS) ||
-      (mBootMode == BOOT_WITH_FULL_CONFIGURATION_PLUS_DIAGNOSTICS) ||
-      (mBootMode == BOOT_IN_RECOVERY_MODE))
-  {
-    if (PlatformConsoles != NULL) {
-      while ((*PlatformConsoles).DevicePath != NULL) {
-        //
-        // Update the console variable with the connect type
-        //
-        if (((*PlatformConsoles).ConnectType & CONSOLE_IN) == CONSOLE_IN) {
-          EfiBootManagerUpdateConsoleVariable (ConIn, (*PlatformConsoles).DevicePath, NULL);
-        }
-
-        if (((*PlatformConsoles).ConnectType & CONSOLE_OUT) == CONSOLE_OUT) {
-          EfiBootManagerUpdateConsoleVariable (ConOut, (*PlatformConsoles).DevicePath, NULL);
-        }
+  if (PlatformConsoles != NULL) {
+    while ((*PlatformConsoles).DevicePath != NULL) {

```

Comment:
nit: Any idea on why we use the `*Pointer.Field` instead of `Pointer->Field`? This is not related to the change itself, so no need to block on it.

Reply:
No. I'd also prefer it to use `->`. I can add that in a separate commit to prevent confusion with the functional change being made.

Reply:
Updated